templates/patterns/social_media_links/social_media_links.ui_patterns.yml

```diff
@@ -0,0 +1,32 @@
+social_media_links:
+  label: "Social media links"
+  description: "Social network icon with link of label."
+  variants:
+    horizontal:
+      label: "Horizontal"
+      description: "It will align the field items horizontally."
```

Comment:
There are no field items here, lets say "Social media links will be arranged horizontally."